lib/ProductOpener/DataQualityFood.pm

```diff
@@ -816,14 +816,27 @@ sub check_nutrition_data ($product_ref) {
 				$nutriments_values_occurences{$nutriment_value} = 1;
 			}
 		}
-		# raise warning if there are 3 or more duplicates in nutriments
+		# raise warning if there are 3 or more duplicates in nutriments and nutriment is above 1
 		foreach my $keys (keys %nutriments_values_occurences) {
-			if ($nutriments_values_occurences{$keys} > 2) {
+			if (($nutriments_values_occurences{$keys} > 2) and ($keys > 1)) {
 				push @{$product_ref->{data_quality_warnings_tags}}, "en:nutrition-3-or-more-values-are-identical";
 				last;
 			}
 		}
 
+		# retrieve max number of occurences
+		my $nutriments_values_occurences_max_value = -1;
+		foreach my $key (keys %nutriments_values_occurences) {
+			if ($nutriments_values_occurences{$key} > $nutriments_values_occurences_max_value) {
+				$nutriments_values_occurences_max_value = $nutriments_values_occurences{$key};
+				# $nutriments_values_occurences_max_key = $key;
+			}
+		}
+		# raise error if all values are identical (this can only apply when they are 0 (because salt or sodium are automatically generated from each others))
```

Comment:
Then what would be the difference with the already existing en:Nutrition all values zero ?

Reply:
I did not see that before, actually. That already existing dq error facet has nothing: https://world.openfoodfacts.org/data-quality-error/en:all-nutrition-values-are-set-to-0
Because:
1) 
```
if ($nid !~ /fruits-vegetables-nuts-estimate-from-ingredients/) {
	if ($product_ref->{nutriments}{$nid} == 0) {
		$nid_zero++;
	}
	else {
		$nid_non_zero++;
	}
}
```
does not exclude "fruits-vegetables-legumes-estimate-from-ingredients_100g"

**comment:** instead of adding this exclusion, that would be better to list which main nutrients we want to select. That way if tomorrow there is a new fruits-something-blabla_100g we will not have problem if we forget to update that condition.

2)
there is a typo: 
```($nid_zero == $nid_n)```
where
**$nid_zero**: number of $nid (nutrient) that contains "_100g" and equal to 0. That number is at most all "_100g" minus fruits-vegetables-nuts-estimate-from-ingredients_100g.
**$nid_n**: all nutrient including "_100g" as well as without "_100g". Example for proteins: "proteins", "proteins_100g", "proteins_unit", "proteins_value".

According to my comment, I am removing this dq facet to replace it by the one implemented in this PR.